src/modules/account/rest-api/account-router.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
+import { Router } from 'express';
+import AccountController from './account-controller';
+import ACCOUNT_ROUTES from './account-routes';
+
+export default class AccountRouter {
+  public static getRoutes(): Router {
+    const router = Router();
+
+    router.post(
+      ACCOUNT_ROUTES.ACCOUNT_ROUTE_ROOT,
+      AccountController.createAccount,
+    );
+    router.get(
```

Comment:
Let's remove this. It may be needed on specific application to find user by username but generally, not